native/src/modules/common/components/LayoutedScrollView.js

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
LayoutContainer is a container. That means this should also be a container. Maybe you can just move this component to the top of withPayloadProvider.js as this is really not that reusable.

Reply:
Moved to `containers`. It is reusable and is reused, see e.g. `SprungbrettContainer`.